src/compas_cgal/straight_skeleton_2.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from compas.geometry import normal_polygon
+from compas.tolerance import Tolerance
```

Comment:
currently you should do `from compas.tolerance import TOL`, which is the global instance of the `Tolerance` class.

i will replace this with a singleton in the future, but currently this would be better...